pkg/openapi/openapi_generated.go

```diff
@@ -10122,15 +10122,7 @@ func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenA
 			Schema: spec.Schema{
 				SchemaProps: spec.SchemaProps{
 					Description: "Duration is a wrapper around time.Duration which supports correct marshaling to YAML and JSON. In particular, it marshals into strings, which can be used as map keys in json.",
-					Properties: map[string]spec.Schema{
```

Comment:
This is outside of the scope of this PR, but there seems to be a problem where this part of the schema varies depending upon whether the build is done in docker or outside of docker.